test/bin/build_images.sh

```diff
@@ -14,29 +14,62 @@ shopt -s nullglob
 SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # shellcheck source=test/bin/common.sh
 source "${SCRIPTDIR}/common.sh"
+# shellcheck source=test/bin/get_crel_version_repo.sh
+source "${SCRIPTDIR}/get_crel_version_repo.sh"
+
+test_envsubst() {
+    local -r file="${1}"
+
+    # Get all variables from a file (expects ${VAR} format, not $VAR)
+    mapfile -t variables < <(awk 'match($0, /(\${.*})/, res) {print res[1]}' "${file}")
+
+    # Iterate over obtained variables and see if current script
+    # has all the values to substitute them.
+    for var in "${variables[@]}"; do
+        value=$(eval echo "${var}")
+        if [ -z "${value}" ]; then
+            echo "Variable ${var} is empty"
+            env
+            # If any of the variables is empty, we don't want to
+            # add this file to build system.
+            return 1
+        fi
+    done
+
+    return 0
+}
 
 configure_package_sources() {
     ## TEMPLATE VARIABLES
-    #
-    # Machine platform type ("x86_64")
-    UNAME_M=$(uname -m)
-    export UNAME_M
+    export UNAME_M                 # defined in common.sh
     export LOCAL_REPO              # defined in common.sh
     export NEXT_REPO               # defined in common.sh
     export BASE_REPO               # defined in common.sh
     export YPLUS2_REPO             # defined in common.sh
+    export CREL_REPO
+
     export SOURCE_VERSION
     export FAKE_NEXT_MINOR_VERSION
     export FAKE_YPLUS2_MINOR_VERSION
     export MINOR_VERSION
     export PREVIOUS_MINOR_VERSION
     export SOURCE_VERSION_BASE
+    export CREL_VERSION
```

Comment:
"CREL" is "current release"?

Reply:
Yes, just like we have "prel" (previous release) in scenarios. Though I guess that might've been a puzzle. I'll expand the abbrev and use `crel` only in filenames